src/aggregate/aggregate_exec.c

```diff
@@ -933,7 +938,7 @@ static int execCommandCommon(RedisModuleCtx *ctx, RedisModuleString **argv, int
     blockedClientReqCtx *BCRctx = blockedClientReqCtx_New(r, blockedClient, spec_ref);
     // Mark the request as thread safe, so that the pipeline will be built in a thread safe manner
     r->reqflags |= QEXEC_F_RUN_IN_BACKGROUND;
-
+    r->qiter.GILTime +=  IsProfile(r) ? (clock() - r->initClock) : 0;
```

Comment:
shouldn't we take the wall-time here as well?
```suggestion
    if (IsProfile(r)) r->qiter.GILTime += clock() - r->initClock;
```